README.md

```diff
@@ -18,7 +18,10 @@ Create projects and choose if they are counted as 'work time'. Select the projec
 **You need to close the application manualy before you shutdown your PC. Otherwise the last running project is not saved completely to database. (will be last state saved by auto-save)**
 
 ### Settings:
-![Settings Screen](readme/images/settings.png?raw=true "Settings")
```

Comment:
can you try to add an icon to each menu entry?

Reply:
Icons added